container-files/build-tensorflow.sh

```diff
@@ -67,39 +71,35 @@ if [ "$(echo "$BUILD_PHASES" | grep dosetup)" != "" ]; then
     apt-get install -y openjdk-8-jdk
     apt-get install -y python3-numpy python3-dev python3-pip python3-wheel
 
-    # Build the sample cuda program "deviceQuery"
-    cd /
-    tar -xvf "$SCRIPTDIR"/device_query.tar.gz
-    #rm "$SCRIPTDIR"/device_query.tar.gz <- don't remove, this is a mounted file
-    cd /usr/local/cuda-9.2/samples/1_Utilities/deviceQuery/
-    make
-    ./deviceQuery
-
     # Hack make python use python3
     ln -s /usr/bin/python3 /usr/bin/python
 
     # Hack, fix assumptions tensorflow build makes about NCCL installation
+    echo ">>>>> Fixing NCCL to comply with tensorflow assumptions."
     mkdir /usr/local/cuda-9.2/nccl
     ln -s /usr/include /usr/local/cuda-9.2/nccl/include
     ln -s /usr/lib/x86_64-linux-gnu /usr/local/cuda-9.2/nccl/lib
     cp "$SCRIPTDIR"/NCCL-SLA.txt /usr/local/cuda-9.2/nccl
 
     # Install bazel using the installer
+    echo ">>>>> Installing Bazel."
     cd /tmp
     chmod +x bazel-$BAZEL_VERSION-installer-linux-x86_64.sh
     ./bazel-$BAZEL_VERSION-installer-linux-x86_64.sh
     rm bazel-$BAZEL_VERSION-installer-linux-x86_64.sh
 
-    # Needed for (apparently) 1.10 and (certainly) 1.11
-    pip3 install keras_applications==1.0.4 --no-deps
-    pip3 install keras_preprocessing==1.0.2 --no-deps
-    pip3 install h5py==2.8.0
+    pip3 install six mock
```

Comment:
Should probably leave a comment with why these are being installed. I've already half-forgotten why this step is being done.

Reply:
Feel free. I have no idea. I just got this from you as far as I recall.